sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -0,0 +1,194 @@
+# Azure Tracing client library for Java
+This package enables distributed tracing across Azure SDK Java libraries through [opentelemetry][opentelemetry]. OpenTelemetry is an open source, vendor-agnostic, single distribution of libraries to provide metrics collection and distributed tracing for services. 
+The Azure core tracing package provides:
+- Context propagation, in order to correlate activities and requests between services with an initial customer action.
+- Tracing user requests to the system, allowing to pinpoint failures and performance issues.
+
+[Source code][source_code] | [API reference documentation][api_documentation] | [Product
+documentation][api_documentation] | [Samples][samples]
+
+## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Maven][maven]
+
+### Adding package to your product
+[//]: # ({x-version-update-start;com.azure:azure-core-tracing-opentelemetry;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-core-tracing-opentelemetry</artifactId>
+  <version>1.0.0-preview.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Default HTTP Client
```

Comment:
Does OpenTelemetry take a dependency on an HTTP client? If not we should remove this section.

Reply:
As discussed offline, users might need to know about this when using this plugin in conjunction with any of our client libraries, so keeping this.